src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -481,7 +481,8 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ov::Model>&
         {ov::op::v8::PriorBox::get_type_info_static(), fuse_type_to_prior_box<ov::op::v8::PriorBox>},
         {ov::op::v0::PriorBoxClustered::get_type_info_static(), fuse_type_to_prior_box<ov::op::v0::PriorBoxClustered>},
         {ov::op::v15::SearchSorted::get_type_info_static(), fuse_type_to_search_sorted_v15},
-        {ov::op::internal::RMS::get_type_info_static(), fuse_type_to_rms}};
+        {ov::op::internal::RMS::get_type_info_static(), fuse_type_to_rms},
+        {ov::DiscreteTypeInfo("InlinedCustomOp", "extension"), wrap_into_original_type}};
```

Comment:
Perhaps we don't need to use this check inside `wrap_into_original_type` for InlinedCustomOp and always apply the conversion regardless of the precisions (precision_map) passed into ConvertPrecision pass

<img width="448" alt="{9912A5D0-F3D8-403A-8293-C0F64C79CB1E}" src="https://github.com/user-attachments/assets/bd655a1e-cddb-492b-a637-b29bcac9ece7" />

but if we just delete the check, each ConvertPrecision call might affect InlinedCustomOp (insert additional converts or like that). could you double check?

Reply:
If we remove it, it can affect other `Assign` and `ReadValue`. Not looks safe.

Reply:
I was thinking about a new `wrap_into_original_type` without this check only for this operation not to affect Assign/ReadValue etc
But right now I believe the current impl is sufficient, don't have an example how to break